hack/import-restrictions.json

```diff
@@ -1,9 +1,228 @@
 [
-   {
-      "baseImportPath" : "pkg/image",
-      "allowedImports" : [
-         "pkg/client",
-         "vendor"
-      ]
-   }
+  {
+    "checkedPackages": [
+      "github.com/openshift/origin/pkg/authorization/apis/authorization",
+      "github.com/openshift/origin/pkg/authorization/apis/authorization/v1"
+    ],
+    "allowedImportPackageRoots": [
+      "vendor/k8s.io/apimachinery",
+      "vendor/github.com/gogo/protobuf"
+    ],
+    "allowedImportPackages": [
+      "vendor/github.com/google/gofuzz",
```

Comment:
Upstream to match what was being written before I stripped vendor from the front of any declarations. IMO it's a little cleaner to look at from a config point of view since you don't really care and there are no nested vendors (right?)